src/java.base/share/classes/sun/launcher/LauncherHelper.java

```diff
@@ -405,12 +405,23 @@ public static void printSystemMetrics() {
         limit = c.getMemoryAndSwapLimit();
         ostream.println(formatLimitString(limit, INDENT + "Memory & Swap Limit: ", longRetvalNotSupported));
 
+        limit = c.getPidsMax();
+        ostream.println(formatLimitString(limit, INDENT + "Maximum number of tasks available to the process: ",
```

Comment:
This uses some different capitalization that for other metrics. Could we perhaps shorten this to: `Maximum Processes Limit:`